src/events/pollVotationResults.js

```diff
@@ -6,15 +6,28 @@ module.exports = {
   async execute(message) {
     const { embeds, author, client, channelId } = message;
 
-    if (author.bot && embeds[0]?.data?.type !== 'poll_result') {
+    if (
+      author.bot &&
+      Array.isArray(embeds[0]?.fields) &&
```

Comment:
My mistake, here we would like to return if embeds[0]?.fields is not an array
```suggestion
      !Array.isArray(embeds[0]?.fields) &&
```